ssv/spectest/tests/runner/consensus/valid_decided_10_operators.go

```diff
@@ -19,8 +19,8 @@ func ValidDecided10Operators() tests.SpecTest {
 				Name:                    "sync committee contribution",
 				Runner:                  testingutils.SyncCommitteeContributionRunner(ks),
 				Duty:                    &testingutils.TestingSyncCommitteeContributionDuty,
-				Messages:                testingutils.SSVDecidingMsgsV(testingutils.TestSyncCommitteeContributionConsensusData, ks, types.BNRoleSyncCommitteeContribution),
-				PostDutyRunnerStateRoot: "a991b6470a8c7a55f4ce89aea91925c2d80a9a8c4258545cc2fb17cabc388719",
+				Messages:                testingutils.SSVDecidingMsgsV(testingutils.TestSyncCommitteeContributionConsensusData(ks), ks, types.BNRoleSyncCommitteeContribution),
+				PostDutyRunnerStateRoot: "ec47a2194bab81891bd581eaf427af211b53e13d1f77a6703eeb56a693caa878",
```

Comment:
this is changed because preconsensus now affects the duty runner state?

Reply:
the runner contains all msg containers that includes messages with preconsensus justifications, yes...
But I don't see in the logic where we add prec justification for this test... will look into it...

Reply:
the commit 66c1585f has this [change](https://github.com/bloxapp/ssv-spec/commit/66c1585ff0a1ef79b31ccd2453a37314cb5a0ea8#diff-28956438e46e71f8b726f9fe106f47ac7cfca29ae303a20aacc6e7837a29b506R21)

This should be overridden by deneb imo and the root may need to change again...
I will resolve conflicts and then regenrate jsons and see what's going on